xrayutilities: new recipe

tschoonj opened the pull request:
<!--
Thank you very much for putting in this recipe PR!

This repository is very active, so if you need help with
a PR or once it's ready for review, please let the right people know.
There are language-specific teams for reviewing recipes.

Currently available teams are:
- python `@conda-forge/help-python`
- python/c hybrid `@conda-forge/help-python-c`
- r `@conda-forge/help-r`
- java `@conda-forge/help-java`
- nodejs `@conda-forge/help-nodejs`
- c/c++ `@conda-forge/help-c-cpp`
- perl `@conda-forge/help-perl`
- Julia `@conda-forge/help-julia`
- ruby `@conda-forge/help-ruby`

If your PR doesn't fall into those categories please contact
the full review team `@conda-forge/staged-recipes`.

Due to GitHub limitations first time contributors to conda-forge are unable
to ping these teams.
Consider asking on our [Gitter channel](https://gitter.im/conda-forge/conda-forge.github.io)
if your recipe isn't reviewed promptly.
-->

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

tschoonj commented:
@dkriegner Would you like to be a maintainer of this recipe?

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes/xrayutilities```) and found it was in an excellent condition.

dkriegner commented:
@tschoonj 
I could be maintainer of this and create pull requests when updates are available. Any other duties I would have?

btw: The current build errors seem to be because setuptools is a runtime dependency. (this was wrong in the [README.md](https://github.com/dkriegner/xrayutilities/commit/ff6bcc51b07f2b74b0a1cebab95ae6f6f17315c9) file!)

tschoonj commented:
@dkriegner I have added you as maintainer. There should be no need to open pull requests yourself when the feedstock is created as the conda-forge bot will open them automatically whenever you upload a new release to PyPI.

I have also added setuptools as run-time dependency
